.golangci.yml

```diff
@@ -96,3 +96,8 @@ issues:
     - linters:
         - lll
       path: "pkg/apis/submarinerconfig/v1alpha1/zz_generated.swagger_doc_generated.go"
+
+    # Replacing "true" with a constant doesn't make code more clear
+    - linters:
+        - goconst
+      text: "string `true` has"
```

Comment:
Please fix the `"true"` mentions in `gateways_controller_test.go` instead of adding an exception.